CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
```

Comment:
Unrelated to these changes, but I find it annoying to have to keep re-adding this after each release. I've filed an issue here proposing improvements, please check it out and +1 it if you agree! https://github.com/getsentry/craft/issues/422